src/main.rs

```diff
@@ -141,35 +140,55 @@ async fn main() -> eyre::Result<()> {
 
     match cli.command {
         Command::Init { lib, api, package } => {
-            let r#type = if lib {
+            let kind = if lib {
                 PackageType::Lib
             } else if api {
                 PackageType::Api
             } else {
                 PackageType::Impl
             };
 
-            command::init(r#type, package).await
+            command::init(kind, package)
+                .await
+                .wrap_err("init command failed")
```

Comment:
Please make them more meaningful eg. "Initialization of `<project-name>` failed

Reply:
This is actually impossible without either a refactor that pulls the project name resolution logic out of `command::init()` and into `main()` or by returning a typed error from the command. I don't think either is worth the trouble so marking as resolved.